streams/fr_groupem6.m3u

```diff
@@ -11,3 +11,11 @@ http://190.2.155.162:8080/m6music/mpegts
 https://shls-tiji-tv-prod-dub.shahid.net/out/v1/98f46736bd8c4404b67e4b7a38cc8976/index.m3u8
 #EXTINF:-1 tvg-id="TiJi.fr",Tiji (1080p)
 https://shls-tiji-tv-prod-dub.shahid.net/out/v1/ee05878a88474f408ff04495d44cc249/index.m3u8
+#EXTINF:-1 tvg-id="M6.fr",M6
+https://origin2-6play.live.6cloud.fr/pool_rnpls9by/m6/sixplayhde/hls_short_fp10_m6/NA/vZNA6hM/index.m3u8
```

Comment:
no sound on this one, the base link gets encrypted data.

Reply:
https://origin2-6play.live.6cloud.fr/pool_rnpls9by/m6/sixplaysd/hls_short_fp10_m6/NA/vBadSAb/index.m3u8 video
https://origin2-6play.live.6cloud.fr/pool_rnpls9by/m6/sixplaysd/hls_short_fp10_m6/NA/aVWvbjp/index.m3u8 audio

Reply:
corriger  les autres de 6play sois sont dead ou drm ?

Reply:
l'une des deux possibilités, aucune idée